src/sentry/static/sentry/app/components/gridEditable/styles.tsx

```diff
@@ -0,0 +1,290 @@
+import styled from 'react-emotion';
+
+import InlineSvg from 'app/components/inlineSvg';
+import space from 'app/styles/space';
+
+const GRID_HEADER_HEIGHT = '45px';
+const GRID_EDIT_WIDTH = '35px';
+const GRID_EDIT_WIDTH_DOUBLE = '70px'; // (2 * GRID_EDIT_WIDTH)
+
+/**
+ * Explanation of z-index:
+ *  - Resizer needs to float above <th> cells to be interactive.
+ *  - Editable needs to float above Resizer to hide the right-most Resizer,
+ */
+const Z_INDEX_RESIZER = '1';
+const Z_INDEX_EDITABLE = '10';
```

Comment:
Should these z-indexes be in the theme module? We've been trying to keep all the zindexes there so it is easier to understand stacking.

Reply:
It shouldn't go into the global theme file as the `grid` container has its own [local stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context#The_example).